string user_text
---
string model_text